### PR TITLE
Up the default timeout for GCStress

### PR DESCRIPTION
**Pull request by jashook**
@RussKeldorph @dotnet/jit-contrib ptal

### CODE REVIEW

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Release gcstress15_pri1r2r

File: netci.groovy [hunk 1]
```diff
@@ -173,6 +173,13 @@ def static genStressModeScriptStep(def os, def stressModeName, def stressModeVar
     if (os == 'Windows_NT') {
         stepScript += "echo Creating TestEnv Script for ${stressModeName}\r\n"
         stepScript += "del ${stepScriptLocation}\r\n"
+         
+        // Timeout in ms, default is 10 minutes. For stress
+        // modes up this to 30 minutes
+        def timeout = 1800000
```
Comment (RussKeldorph) [inline netci.groovy]: I would just inline this to keep it consistent with runtest.cmd, but it's not a big deal.

**Comment by RussKeldorph** [comment]
LGTM

**Pull request merged**